lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -180,11 +182,16 @@ public APIGatewayProxyResponseEvent handleRequest(
                     ipvSessionService.generateIpvSession(
                             clientOAuthSessionId, null, emailAddress, false);
 
+            var gvcsAccessTokenClaim = getGvcsAccessTokenClaim(claimsSet);
+            if (gvcsAccessTokenClaim.isPresent()) {
+                validateEvcsAccessToken(gvcsAccessTokenClaim.get(), claimsSet);
+            }
```

Comment:
Can we put this behind the feature flag as well - we've had a couple of incidents related to parsing claims from orchestration, so I think we need to be particularly careful in this area.